addons/web/static/src/scss/bs_mixins_overrides.scss

```diff
@@ -7,6 +7,30 @@
 // otherwise we will have deprecation messages for assets_common generation
 $enable-deprecation-messages: false !default;
 
+// Override color-contrast function to handle the alpha component of colors
+@function color-contrast($color,$background:$body-bg,$color-contrast-dark: $color-contrast-dark, $color-contrast-light: $color-contrast-light, $min-contrast-ratio: $min-contrast-ratio){
+
+    $real-color: opaque($background, $color);
```

Comment:
You should not change the signature of the bootstrap method you override (at best you can add new arguments at the end but not change the meaning of the existing ones). So here I suggest you keep the first argument name to be "background" and add an extra argument at the end named "main-background".